### ISSUE TITLE
Is it possible to enable "Remote Browser" for Capybara?

### ISSUE THREAD

**Issue by William-Yeh** [elliterate/capybara.py#17]
In Ruby version of Capybara, it is possible to connect to remote browser (such as ChromeDriver).

For example: https://gist.github.com/jasoncodes/6025475
```ruby
Capybara.register_driver :chrome do |app|
    Capybara::Selenium::Driver.new(app,
      :browser => :remote,
      :desired_capabilities => :chrome,
      :url => "http://#{ENV['CHROME_HOSTNAME']}:4444/wd/hub"
    )
```

Is it possible to modify the `capybara.py/capybara/selenium/browser.py` to add a `"remote"` case for  `browser_name`?

Thanks.

**Comment by William-Yeh** [comment]
I've tested and sent a PR: https://github.com/elliterate/capybara.py/pull/18

**Comment by elliterate** [comment]
Resolved by #18.
